Intersect.Editor/Forms/Editors/Events/CommandPrinter.cs

```diff
@@ -1085,7 +1085,7 @@ private static string GetCommandText(PlayAnimationCommand command, MapInstance m
                 return Strings.EventCommandList.playanimation.ToString(
                     AnimationBase.GetName(command.AnimationId),
                     Strings.EventCommandList.animationonmap.ToString(
-                        Strings.EventCommandList.mapnotfound, command.X, command.Y, Strings.Directions.dir[command.Dir]
+                        Strings.EventCommandList.mapnotfound, command.X, command.Y, Strings.Direction.dir[(Direction)command.Dir]
```

Comment:
On the other lines you consume command.Dir directly, why is this cast needed?

Reply:
https://github.com/AscensionGameDev/Intersect-Engine/pull/1708#discussion_r1126089149